core/src/main/scala/kafka/coordinator/group/GroupCoordinatorAdapter.scala

```diff
@@ -511,4 +532,53 @@ class GroupCoordinatorAdapter(
 
     future
   }
+
+  override def partitionFor(groupId: String): Int = {
+    coordinator.partitionFor(groupId)
+  }
+
+  override def onTransactionCompleted(
```

Comment:
Curious on the rationale for renaming these two. Did you just not agree with the old group coordinator names?

Reply:
I found the old name a bit weird.